CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Haversine formula and AbstractSphericalDistance [#588](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/588)
```

Comment:
Oh and we've set this repo up so that every pull request is required to change our changelog, so that we have a little log and what's been introduced/changed in what version